lib/ui/common/app_backdrop.dart

```diff
@@ -18,7 +18,7 @@ class AppBackdrop extends StatelessWidget {
     final double normalStrength = clampDouble(strength, 0, 1);
     if (settingsLogic.useBlurs) {
       return BackdropFilter(
-        filter: ImageFilter.blur(sigmaX: normalStrength * 5.0, sigmaY: normalStrength * 5.0),
+        filter: ImageFilter.blur(sigmaX: normalStrength * 15.0, sigmaY: normalStrength * 15.0),
```

Comment:
This is pretty expensive at 15 passes, consider lowering it to avoid jank on older devices.